docs/resources/meeting_conferences.md

```diff
@@ -0,0 +1,526 @@
+    interval   = 2
+    points     = [1, 5]
+  }
+
+  configuration {
+    is_send_notify   = true
+    is_send_calendar = true
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `account_name` - (Optional, String, ForceNew) Specifies the (HUAWEI Cloud meeting) user account name to which the
+  meeting initiator belongs. Changing this parameter will create a new resource.
+
+* `account_password` - (Optional, String, ForceNew) Specifies the user password to which the meeting initiator belongs.
+  Required if `account_name` is set. Changing this parameter will create a new resource.
+
+* `app_id` - (Optional, String, ForceNew) Specifies the ID of the Third-party application.
+  Changing this parameter will create a new resource.
+
+  -> You can apply for an application and obtain the App ID and App Key in the console.
+
+* `app_key` - (Optional, String, ForceNew) Specifies the Key information of the Third-party APP.
+  Required if `app_id` is set. Changing this parameter will create a new resource.
+
+* `corp_id` - (Optional, String, ForceNew) Specifies the corporation name to which the meeting initiator belongs.
+  Required if the application is used in multiple enterprises. Only availabe if `app_id` is set.
+  Changing this parameter will create a new resource.
+
+* `user_id` - (Optional, String, ForceNew) Specifies the user ID of the meeting initiator.
+  Only availabe if `app_id` is set. Changing this parameter will create a new resource.
+
+-> Exactly one of account authorization and application authorization you must select.
+
+* `topic` - (Required, String) Specifies the conference topic. The topic can contain `1` to `128` characters.
+
+* `meeting_room_id` - (Required, String) Specifies the cloud meeting room ID.
+
+* `duration` - (Optional, Int) Specifies the duration of the conference.
+  The valid value is range from `15` to `1440`, defaults to `30`.
+
+* `start_time` - (Optional, String) Specifies the conference start time (UTC time).
+  The time format is `YYYY-MM-DD hh:mm`, e.g. `2006-01-02 15:04`.
+
+  -> If you want to start a conference at `08:00` (UTC+8), you need to specified the time with `00:00`.
+
+* `media_types` - (Optional, List) Specifies the conference media type list.
+  It consists of one or more enumerations, and the valid values are as follows:
+  + **Voice**: Voice.
+  + **Video**: SD video.
+  + **HDVideo**: High-definition video (mutually exclusive with Video, if Video and HDVideo are selected at the same
+    time, the system will select Video by default).
+  + **Data**: Multimedia (If omitted, the system configuration will determines whether to automatically add **Data**).
+
+* `is_auto_record` - (Optional, Int) Specifies whether the conference automatically starts recording, it only takes
+  effect when the recording type is:
+  + **1**: Automatically start recording.
+  + **0**: Do not start recording automatically.
+
+  The default is **0** (not to start automatically).
```

Comment:
"Defaults to **0**" or "The default value is **0**"

Reply:
done.